submissions/mustbefail/oop/index.js

```diff
@@ -0,0 +1,88 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, hands, legs, friends, phrase) {
+    this.friends = friends;
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.hands = hands;
```

Comment:
The point of OOP is that:
- a super class has only shared properties
- a child class introduces and handles properties unqiue to that child class

I.e. `hands` is smth unique to humans. Cats or dogs shouldn't have any hands among their props while they inherit them as a property from class `Inhabitant`.